tests/Webapi/Webapi__Url__test.re

```diff
@@ -0,0 +1,10 @@
+open Webapi.Url;
+
+let params = URLSearchParams.make("key1=value1&key2=value2");
+URLSearchParams.forEach(
+   (value, key) => {
+      print_endline(value);
```

Comment:
You can use `Js.log2(value, key)` here to more closely follow the example in https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams/forEach#Examples

Reply:
I think this will work, actually:

```reason
URLSearchParams.forEach(Js.log2, params);
```

Reply:
Yeah I originally had `Js.log2` in there but went for `print_endline` to be more "Reason-y". But this is BuckleScript land so `Js.log2` makes more sense. And yep, that'll work perfectly. Pushing up now.